tests/suites/test_suite_x509parse.data

```diff
@@ -1306,9 +1306,9 @@ X509 CRT ASN1 (TBS, SubPubKeyInfo missing)
 depends_on:MBEDTLS_RSA_C:MBEDTLS_SHA256_C
 x509parse_crt:"30693054a0030201008204deadbeef300d06092a864886f70d01010b0500300c310a30080600130454657374301c170c303930313031303030303030170c303931323331323335393539300c310a30080600130454657374300d06092a864886f70d01010b0500030200ff":"":MBEDTLS_ERR_PK_KEY_INVALID_FORMAT + MBEDTLS_ERR_ASN1_OUT_OF_DATA
 
-X509 CRT ASN1 (TBS, inv SubPubKeyInfo, inv tag
+X509 CRT ASN1 (TBS, inv SubPubKeyInfo, inv tag)
```

Comment:
This should be moved to commit "Fix test case name formatting in X.509 parsing suite", shouldn't it? It doesn't have to do with "Use ASN.1 NULL TLVs when testing invalid tags ".

Reply:
Yes, that's cleaner.